packages/migrations/src/testnet_migrations.ts

```diff
@@ -148,22 +252,19 @@ export async function runMigrationsAsync(supportedProvider: SupportedProvider, t
     );
 
     logUtils.log('Configuring Exchange...');
-    await exchange.setProtocolFeeCollectorAddress.awaitTransactionSuccessAsync(stakingProxy.address);
-    await exchange.setProtocolFeeMultiplier.awaitTransactionSuccessAsync(new BigNumber(150000));
-    await exchange.registerAssetProxy.awaitTransactionSuccessAsync(deployedAddresses.erc20Proxy);
-    await exchange.registerAssetProxy.awaitTransactionSuccessAsync(deployedAddresses.erc721Proxy);
-    await exchange.registerAssetProxy.awaitTransactionSuccessAsync(deployedAddresses.erc1155Proxy);
-    await exchange.registerAssetProxy.awaitTransactionSuccessAsync(deployedAddresses.multiAssetProxy);
-    await exchange.registerAssetProxy.awaitTransactionSuccessAsync(deployedAddresses.staticCallProxy);
     await exchange.transferOwnership.awaitTransactionSuccessAsync(governor.address);
     logUtils.log('Exchange configured!');
 
+    logUtils.log('Configuring ERC20BridgeProxy...');
+    await erc20BridgeProxy.transferOwnership.awaitTransactionSuccessAsync(governor.address);
+    logUtils.log('ERC20BridgeProxy configured!');
+
     logUtils.log('Configuring ZrxVault...');
-    await zrxVault.addAuthorizedAddress.awaitTransactionSuccessAsync(governor.address);
+    await zrxVault.transferOwnership.awaitTransactionSuccessAsync(governor.address);
```

Comment:
Nit: I think we'll still want to add the `governor` as an authorized address. A bunch of functions including `setStakingProxy` and `enterCatastrophicFailureMode` are `onlyAuthorized`. 

Is there a reason why it shouldn't be authorized?

Reply:
These were moved inside the `ZeroExGovernor` `functionCalls` array to keep all of the configurations in one place. Doesn't really matter which we end up doing